R/manip.r

```
@@ -66,16 +66,18 @@ filter_ <- function(.data, ..., .dots = list(), .preserve = TRUE) {
 
 #' Select rows by position
 #'
+#' Select rows by their ordinal position in the tbl.  Grouped tbls use
+#' the ordinal position within the group.
+#'
 #' Slice does not work with relational databases because they have no
 #' intrinsic notion of row order. If you want to perform the equivalent
 #' operation, use [filter()] and [row_number()].
 #'
-#' Positive values select rows to keep; negative values drop rows. The
-#' values provided must be either all positive or all negative.
-#'
 #' @family single table verbs
 #' @param .data A tbl.
-#' @param ... Integer row values.
+#' @param ... Integer row values.  Provide either positive values to keep,
+#'   or negative values to drop. The values provided must be either all
+#'   positive or all negative.
```

Mention what happens if the row doesn't exist?

---

Wasn't sure whether that behavior was set yet - looks like it silently ignores indices outside the range of data, is that what we should document?